src/drawers/drawer.ts

```diff
@@ -67,8 +69,17 @@ export module Drawers {
      *
      * @param{any[]} data The data to be drawn
      */
-    protected _enterData(data: any[]) {
-      // no-op
+    private _setSelectionData(data: any[]) {
```

Comment:
maybe `bindSelectionData()`? "set" seems like it should be reserved for simple setters.